src/main/resources/application.properties

```diff
@@ -21,6 +21,8 @@ quarkusio.localized.cn.web-uri=https://cn.quarkus.io/
 quarkusio.localized.cn.git-uri=https://github.com/quarkusio/cn.quarkus.io.git
 quarkusio.localized.ja.web-uri=https://ja.quarkus.io/
 quarkusio.localized.ja.git-uri=https://github.com/quarkusio/ja.quarkus.io.git
+# We do not want to show the search index page in production as it is there only for testing purposes:
+%prod.search.ui.enabled=false
```

Comment:
hmm 

> %staging.quarkus.config.profile.parent=prod

do we need to override the value for the property then ?:
%staging.search.ui.enabled=true

Reply:
yep!